internal/pkg/plugin/githubactions/golang/templates.go

```diff
@@ -23,14 +23,14 @@ jobs:
       with:
         go-version: 1.17
     - name: Test
-      run: [[- if not .Test.Command]] go test ./...[[- else]][[.Test.Command]][[- end]] [[- if .Test.Coverage.Enable]] -race -covermode=atomic -coverprofile=[[- if not .Test.Coverage.Output]]coverage.out[[- else]][[.Test.Coverage.Output]][[- end]] [[- end]]
+      run: [[if not .Test.Command]] go test ./...[[- else]][[.Test.Command]][[- end]] [[- if .Test.Coverage.Enable]] -race -covermode=atomic -coverprofile=[[- if not .Test.Coverage.Output]]coverage.out[[- else]][[.Test.Coverage.Output]][[- end]] [[- end]]
     [[- if .Test.Coverage.Enable]]
     - name: comment PR
       uses: machine-learning-apps/pr-comment@master
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
Do not change the name. secrets.GITHUB_TOKEN already exists by default.

Same for all the followings.
https://docs.github.com/en/actions/security-guides/automatic-token-authentication